mex/backend/exceptions.py

```diff
@@ -34,23 +34,35 @@ class ErrorResponse(BaseModel):
     debug: DebuggingInfo
 
 
-def handle_uncaught_exception(request: Request, exc: Exception) -> JSONResponse:
+def handle_validation_error(request: Request, exc: Exception) -> Response:
+    """Handle pydantic validation errors and provide debugging info."""
+    logger.exception("ValidationError %s", exc)
+    return Response(
+        content=ErrorResponse(
+            message=str(exc),
+            debug=DebuggingInfo(
+                errors=[
+                    jsonable_encoder(e) for e in cast(ValidationError, exc).errors()
+                ],
+                scope=DebuggingScope.model_validate(request.scope),
+            ),
+        ).model_dump_json(),
+        status_code=status.HTTP_400_BAD_REQUEST,
```

Comment:
you're right, i re-raised those validation errors and then let them rip uncaught, resulting in a 500. because if there is invalid data in the db, it should escalate.

-> [3dee885](https://github.com/robert-koch-institut/mex-backend/pull/168/commits/3dee885a4fd6dfb24fae27e38c3ce99b7d337bd2)